blog/2023-10-7-Blockchain/index.md

```diff
@@ -0,0 +1,145 @@
+---
+slug: intro-to-blockchain
+title: 'Blockchain - Future of Tommorow'
+authors:
+  - name: Kashish Singh
+    title: Technical Team Member [TSDW-TCET]
+    url: https://github.com/kashish-51
+    image_url: 	https://avatars.githubusercontent.com/u/136115355?…00&u=e75a3214ca24205e70ac923046f3d68e209c0cfc&v=4
+
+tags: [Blockchain, Data, Software, Technology ]
+description: 'Blockchain - Future of Tommorow'
+keywords: [Blockchain, Data, Software, Technology, Bitcoin ]
+---
+
+
+> Ever wondered what blockchain is and how it looks? Let's explore the realm of blockchain together. This blog focuses on the basics of blockchain technology along with its applications. In this evolving era, it is important to understand this technology because, according to  recent research, blockchain has the potential to reshape the future and the way we perceive it.
+
+<br />
+
```

Comment:
Recommend [utilizing truncate](https://opensource.tcetmumbai.in/blog/blog-posting-guidelines#utilizing-truncate) to cut off the blog as it is overflowing the blogs page. Follow the posting guidelines on how to implement it.

Reply:
Added truncate